Commands/Markdown.cs

```diff
@@ -24,7 +24,19 @@ public class Markdown : ApplicationCommandModule
                 return;
             }
 
-            message = await ctx.Channel.GetMessageAsync(messageId);
+            DiscordChannel channel;
+            channel = await ctx.Client.GetChannelAsync(ctx.Channel.Id);
```

Comment:
This is redundant and just makes the bot send a request to fetch a channel it already has in memory (`ctx.Channel`)